components/dictionary.txt

```diff
@@ -1,11 +1,133 @@
+actin
+ADT
+al
 al.
+AML
+AnaLysis
+ASXL
+Azacytidine
+arounds
```

Comment:
Convince me this is correct?
```suggestion
```

Reply:
Ugh this is because of its problem with hyphens. I wanted to put here `work-arounds` but it would not work.